client/extensions/woocommerce/app/settings/email/email-settings/components/customer-notification.js

```diff
@@ -15,7 +15,7 @@ import FormSettingExplanation from 'components/forms/form-setting-explanation';
 import ListItem from 'woocommerce/components/list/list-item';
 import ListItemField from 'woocommerce/components/list/list-item-field';
 
-const CustomerNotification = ( { item, checked, onChange, isPlaceholder } ) => {
+const CustomerNotification = ( { item, checked, onChange, loading } ) => {
```

Comment:
It looks like you're more interested in ! loading than in loading, so please consider having loaded as the prop instead

Also please add whichever you choose to the propTypes for this component.